### PR TITLE
NVyrsta

### PR DESCRIPTION
**Pull request by NVyrsta**
- [DEMO LINK](https://NVyrsta.github.io/js_get_data_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+// Variables
+const populationData = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+// Calculation
+const populationList = [...populationData].map(data =>
+  Number(data.innerText.replaceAll(',', '')));
+const total = populationList.reduce((amount, population) =>
+  amount + population, 0);
```
Comment (Avichez) [inline src/scripts/main.js:12]: You can combine these two methods into one reduce method.
```suggestion
const total = [...populationData].reduce((amount, population) => {
  const numberPopulation = Number(population.innerText.replaceAll(',', '')));
  
  return amount + numberPopulation;
  }, 0);
```

**Review by Avichez — APPROVED** [review]
Great and clear solution 👍
Added just one comment

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
 
-// write your code here
+// Variables
+const populationData = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+// Calculation
+const populationList = [...populationData].map(data =>
+  Number(data.innerText.replaceAll(',', '')));
```
Comment (alexandra-protyanova) [inline src/scripts/main.js:10]: 🔥

**Review by alexandra-protyanova — APPROVED** [review]
Good job 👍

File: src/scripts/main.js (hunk 1 above)
Comment (NVyrsta) [reply src/scripts/main.js:12]: Thanks :)

**Pull request closed without merging**